app/renderer/tasks_handler.js

```diff
@@ -60,7 +61,7 @@ class TasksHandler {
     _parseData(data) {
         return data.map((suite) => {
             let result = new Suite(suite.title);
-            result.tasks = suite.tasks.map((task) => new Task(task.title, task.path, task.command));
+            result.tasks = suite.tasks.map((task) => new Task(task.title, task.path, task.command, AppStatus.config));
```

Comment:
Having a duplicate of AppStatus config on each task is **not** a good idea.

Each task class should only contain the data required for itself

Reply:
you are right.